cmake/Modules/FindGCSSDK_EP.cmake

```diff
@@ -63,7 +63,12 @@ if (NOT TILEDB_FORCE_ALL_DEPS OR TILEDB_GCSSDK_EP_BUILT)
   endif()
 endif()
 
-if (NOT GCSSDK_FOUND)
+if (NOT TILEDB_GCS_CLASSIC)
```

Comment:
Why do we need to keep the old way?

Reply:
Because we have [battle scars](https://github.com/conda-forge/tiledb-feedstock/commit/6a74e0103b7ef#diff-d7075654874cb08007a21aaab3ecd4b3453a9087e7505d034d548b8938b599bcR11)™️. (don't worry, we'll remove the old thing once we know the new thing works everywhere).